### ISSUE TITLE
Executing 3llo fails after install

### ISSUE THREAD

**Issue by emilyaviva** [qcam/3llo#4]
I'm on Mac OS X Sierra (10.12.3), with ruby 2.4.0(p0) installed via Homebrew. (I also have rbenv installed, but I'm in my home directory where there is no particular ruby specified.) I installed 3llo via `gem install 3llo`, set my environment variables, but then on execution, I get the following error:
```
/usr/local/Cellar/ruby/2.4.0/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- bundler/setup (LoadError)
	from /usr/local/Cellar/ruby/2.4.0/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/lib/ruby/gems/2.4.0/gems/3llo-0.1.7/bin/3llo:3:in `<top (required)>'
	from /usr/local/bin/3llo:22:in `load'
	from /usr/local/bin/3llo:22:in `<main>'
```
Can anyone shed some light on why this might be happening?

**Comment by qcam** [comment]
maybe you can try `sudo gem install 3llo`?

**Comment by emilyaviva** [comment]
You shouldn't need to install a gem with root powers for it to work for your local user. For example, I simply ran `gem install sass` and that worked just fine.

**Comment by qcam** [comment]
@emilyaviva let me try if I could replicate that on my computer then.

**Comment by qcam** [comment]
@emilyaviva hey the bug should be fixed in version 0.1.8 with this commit https://github.com/qcam/3llo/commit/1225dbc27eea4310756f2cc138db9d4cb4ef0e8f. Can you please try again to see if it works now?

**Comment by emilyaviva** [comment]
It works! Marvelous!! Thank you. 🥇
